iam/__tests__/communitystaking.test.ts

```diff
@@ -0,0 +1,248 @@
+// ---- Test subject
+import { RequestPayload } from "@gitcoin/passport-types";
+import {
+  CommunityStakingBronzeProvider,
+  CommunityStakingSilverProvider,
+  CommunityStakingGoldProvider,
+  stakingSubgraph,
+  DataResult,
+} from "../src/providers/communitystaking";
+
+// ----- Libs
+import axios from "axios";
+
+jest.mock("axios");
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+const MOCK_ADDRESS = "0xcF314CE817E25b4F784bC1f24c9A79A525fEC50f";
+const MOCK_ADDRESS_LOWER = MOCK_ADDRESS.toLowerCase();
+
+const generateSubgraphResponse = (address: string, total: string): DataResult => {
+  return {
+    data: {
+      data: {
+        address: address,
+        users: [
+          {
+            xstakeAggregates: [
+              {
+                total: total,
+                round: {
+                  id: "2",
+                },
+              },
+            ],
+          },
+        ],
+      },
+    },
+  };
+};
+
+const invalidCommunityStakingResponse = {
+  data: {
+    data: {
+      users: [{}],
+    },
+  },
+};
+
+interface RequestData {
+  query: string;
+}
+
+describe("Attempt verification", function () {
```

Comment:
I would also add a test in this suite for: `handles invalid verification attempt where an exception is thrown` because axios seems to throw on errors.